crates/layout/src/proto/layout_style.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::layout_style;
+
+use super::LayoutStyle;
+
+impl TryFrom<LayoutStyle> for layout_style::LayoutStyle {
+    type Error = dc_proto::Error;
+
+    fn try_from(proto: LayoutStyle) -> Result<Self, Self::Error> {
```

Comment:
hey, this came out pretty nicely!

Reply:
Yeah, TryFrom seemed to be the way to go.